ocis-pkg/shared/errors.go

```diff
@@ -85,3 +85,11 @@ func MissingServiceAccountSecret(service string) error {
 		"the config/corresponding environment variable).",
 		service, defaults.BaseConfigPath())
 }
+
+func MissingWOPISecretError(service string) error {
```

Comment:
the wopi secret is a different thing then our internal jwt secret. so we need a seperate error for it. IIRC this becomes relevant when using the office365 proxy.